lib/msal-node/src/client/IPublicClientApplication.ts

```diff
@@ -51,4 +52,10 @@ export interface IPublicClientApplication {
 
     /** Clear the cache */
     clearCache(): void;
+
+    /** Gets all cached accounts */
+    getAllAccounts(): Promise<AccountInfo[]>;
+
+    /** Removes cache artifacts associated with the given account */
+    signOut(request: SignOutRequest): Promise<void>;
```

Comment:
For later reference, should we call the `logOut` API in `msal-browser` also `signOut`?

Reply:
Sure. Called it out for vFuture. Not a blocker for this PR.